src/Tasks/Microsoft.NET.Build.Tasks/ResolvePublishAssemblies.cs

```diff
@@ -83,7 +83,7 @@ protected override void ExecuteCore()
                 TaskItem item = new TaskItem(resolvedAssembly.SourcePath);
                 item.SetMetadata("DestinationSubPath", resolvedAssembly.DestinationSubPath);
                 item.SetMetadata("AssetType", resolvedAssembly.Asset.ToString().ToLower());
-                item.SetMetadata(MetadataKeys.PackageName, resolvedAssembly.Package.Id.ToString().ToLower());
+                item.SetMetadata(MetadataKeys.PackageName, resolvedAssembly.Package.Id.ToString());
```

Comment:
Cool ... It's a UX issue if the files have lowercase package names. Some devs are bound to wonder: Do they always need to be lowercase in the *artifact.xml* file? This will fix that prob.